### ISSUE TITLE
tN word count accuracy

### ISSUE THREAD

**Issue by mandolyte** [unfoldingWord-box3/book-package#10]
For Psalm 1, the number of notes 565 and the word count is only 28. 

```
Translation Notes for "PSA" and Chapters 1
Total number of notes: 565
Distinct number of words in notes: 23
Total number of words in the notes: 28
```

**Comment by mandolyte** [comment]
Found that the chapter filter was using the wrong column. Per tests it is now close. 

Should be:
```
Passed: en_tn_19-PSA_occnotes.txt  Total: 769  Distinct: 222
```

Actual is now closer:
```
Translation Notes for "PSA" and Chapters 1
Total number of notes: 22
Distinct number of words in notes: 225
Total number of words in the notes: 762
```

**Comment by mandolyte** [comment]
Results now match tests. Found that when aggregating the text in the spreadsheet cells, that I was simply concatenating. This meant that a word at the end of line would be joined to the first word of next line. To fix I only concatenated with a space: `line1 + " " + line2` instead of `line1+line2`

**Comment by mandolyte** [comment]
Starting point:
```
Translation Notes for "MRK" and Chapters (ALL)
Total number of notes: 1428
Total Word Count: 2009
Unique Words: 581
```
My test/verification code shows:
```
$ nx en_tn_42-MRK_occnotes.mjs
(node:11584) ExperimentalWarning: The ESM module loader is experimental.
Failed: en_tn_42-MRK_occnotes.txt
--- Expected total of 769 and distinct of 222
--- Found total of 31202 and distinct of 2574
--- See word list found at en_tn_42-MRK_occnotes.txt.allwords.txt
$
```
This is a huge difference. Suspect something is amiss in the row selection...

**Comment by mandolyte** [comment]
Pushed a change which now matches my tests. The change made was switch the delimiter used to aggregate the text from a space to a newline. I'm a bit uncomfortable with this. It may imply that I have regex operator that is being greedy. If so this means that the counts may still be too low.

Component now shows for Mark:
```
Translation Notes for "MRK" and Chapters (ALL)
Total number of notes: 1427
Total Word Count: 32290
Unique Words: 2655
```

**Comment by mandolyte** [comment]
a) found juxtaposed words with an ellipse "..." character. This meant that the words were treated as a single word. Fixed by replacing such characters with a blank.
b) by adjusting the order of the regexes I found that entire sentences were being removed from consideration. While this appears to be a greedy issue, I could not find the reason. By ordering the regexes into what I hope is the most narrow specific cases to most general, this seemed to fix the problem.

Counts now show:
```
Translation Notes for "MRK" and Chapters (ALL)
Total number of notes: 1427
Total Word Count: 32596
Unique Words: 2630
```
Note that the issue with the ellipse means the unique count will decrease slightly.
